tools/underlay.repos

```diff
@@ -22,4 +22,4 @@ repositories:
   ompl/ompl:
     type: git
     url: https://github.com/ompl/ompl.git
-    version: 1.5.0
+    version: main
```

Comment:
Why not use a specific release? I'd really like to not rebuild ompl breaking the cache every commit to its master branch. The ompl builds take 30-40 minutes and make our CI not that fast when we have to build it

Reply:
Ok, v1.5.2 still includes the recent ament changes

```suggestion
    version: 1.5.2
```

Reply:
Should note that normally, only the dockerhub image build would be building underlay, not sure how frequently other PRs have been changing the `underlay.repos` file.

Reply:
Good point, I’m ok with this. I see it occasionally but its not very often.